playwright/pages/shippingMethodsPage.ts

```diff
@@ -5,12 +5,11 @@ import { AssignCountriesDialog } from "@pages/dialogs/assignCountriesDialog";
 import { RightSideDetailsPage } from "@pages/pageElements/rightSideDetailsSection";
 import type { Page } from "@playwright/test";
 
-export class ShippingMethodsPage {
-  readonly page: Page;
-  readonly basePage: BasePage;
-  readonly rightSideDetailsPage: RightSideDetailsPage;
-  readonly assignCountriesDialog: AssignCountriesDialog;
-  readonly deleteShippingMethodDialog: DeleteShippingMethodDialog;
+export class ShippingMethodsPage extends BasePage {
+  basePage: BasePage;
```

Comment:
question: Do you need basePage props if you extends BasePage class?

Reply:
@poulch fixed